### DISCUSSION TITLE
Standardizing radon sensor units

### DISCUSSION THREAD

**Discussion by Kirchoff** [home-assistant/architecture#999] [Entity Models]
As requested from @bdraco, I'd like to propose a discussion on how to handle making radon measuring sensors a standard. My PR that prompted this discussion is located here: #103709

My personal interest in this topic is that I recently added an AirThings View Plus air quality monitor to my home. While I can view the radon measurements in the AirThings app with either SI (Bq/m^3) or Imperial (pCi/L) units, the Home Assistant integration defaults to SI. It also does not support viewing those sensors in Imperial units. I noticed that there are a total of 3 existing components that measure/display radon, and all 3 do it quite differently. This PR was an attempt to standardize. 

Please let me know if I can add anything further. Thanks.

**Comment by jpbede** [comment]
Isn't this a duplicate of https://github.com/home-assistant/architecture/discussions/886?

**Reply by Kirchoff** [reply]
It sure looks like it is, my apologies for missing that.

**Reply by bdraco** [reply]
@Kirchoff It would be good to comment on that discussion with any thoughts as it seems a bit stalled due to lack of interest.  I suspect the number of people who are experts on radiation measurements is a small group which makes it a bit more difficult to arrive at a conclusion. These type of discussions can take a long time because its very hard to change later if we get it wrong.
